pkgs/development/python-modules/crytic-compile/default.nix

```diff
@@ -2,10 +2,14 @@
 
 buildPythonPackage rec {
   pname = "crytic-compile";
-  version = "0.1.9";
+  version = "0.1.12";
 
   disabled = pythonOlder "3.6";
 
+  patchPhase = ''
+    substituteInPlace setup.py --replace 'version="0.1.9",' 'version="0.1.12",'
```

Comment:
```suggestion
    substituteInPlace setup.py --replace 'version="0.1.9",' 'version="${version}",'
```

Reply:
Maybe also create an upstream issue that they fix this.

Reply:
it's already fixed there, so shouldn't be needed in the next version